### PR TITLE
Fill

### PR DESCRIPTION
**Pull request by wooorm**
Here’s some useful stuff!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+The ISC License
```
Comment (RichardLitt) [inline LICENSE:1]: Why ISC and not MIT?

Comment (wooorm) [reply LICENSE:1]: Cause it said so in the ~~readme~~package.json (I think)! 🤷‍♂️

File: README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Or use it on the CLI directly
+
+```sh
+remark -u standard-readme-preset README.md
+```
+
+Or use this on the API:
+
+```diff
```
Comment (RichardLitt) [inline README.md:45]: Huh! What does `diff` mean for highlighting with linguist?

**Review by RichardLitt — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (wooorm) [reply README.md:45]: Adds a cute green background to the added line!

File: LICENSE (hunk 1 above)
Comment (RichardLitt) [reply LICENSE:1]: I think that's npm's default. No idea why, really. Will look into that.

**Pull request merged**